library_website/settings/base.py

```diff
@@ -33,11 +33,9 @@
     'django.contrib.messages',
```

Comment:
Am I correct that this file (`base.py`) and `staff/models.py` only have linting changes? If that's the case could we revert these? It will help cut down on some of the unrelated noise and troubleshoot the problem with unit tests. If I'm wrong about that, please disregard.

Reply:
Yes; that's right.  They used to have substantive changes, but then I took them out, and I guess the linting changes are what's left.  Reverting them now.

Reply:
Ok; those should be back to normal.